Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -619,15 +619,11 @@ public virtual void ServerChangeScene(string newSceneName, LoadSceneMode sceneMo
             // Let server prepare for scene change
             OnServerChangeScene(newSceneName);
 
-            LoadSceneParameters loadSceneParameters = new LoadSceneParameters(sceneMode, physicsMode);
-
-            loadingSceneAsync = SceneManager.LoadSceneAsync(newSceneName, loadSceneParameters);
+            loadingSceneAsync = SceneManager.LoadSceneAsync(newSceneName);
 
             SceneMessage msg = new SceneMessage()
             {
                 sceneName = newSceneName,
-                sceneMode = loadSceneParameters.loadSceneMode,
-                physicsMode = loadSceneParameters.localPhysicsMode
```

Comment:
why was physicsMode removed from this PR?
wasn't this important?

Reply:
Not to the client.  That was an oops on my part when I put that in months ago.